src/modules/item/sagas.spec.ts

```diff
@@ -201,6 +202,25 @@ describe('when handling the save item request action', () => {
           .dispatch(saveItemRequest(item, contentsToSave))
           .run({ silenceTimeout: true })
       })
+
+      it('should put a fetch item curation request action if the item is a TP one', () => {
+        return expectSaga(itemSaga, builderAPI, builderClient)
+          .put(fetchItemCurationRequest(item.collectionId!, item.id))
+          .dispatch(
+            saveItemSuccess(
+              { ...item, urn: 'urn:decentraland:mumbai:collections-thirdparty:thirdparty2:one-third-party-collection' },
+              contentsToSave
+            )
+          )
+          .run({ silenceTimeout: true })
+      })
+
+      it('should not put a fetch item curation request action if the item is a standard one', () => {
+        return expectSaga(itemSaga, builderAPI, builderClient)
+          .not.put(fetchItemCurationRequest(item.collectionId!, item.id))
+          .dispatch(saveItemSuccess(item, contentsToSave))
+          .run({ silenceTimeout: true })
+      })
```

Comment:
I would split them into another describe:
```
describe('when handling the save item success action', () => ...)
```
As this section describes the `as handling the save item request action`

Reply:
Good idea, updated!